cmd/checkmarxExecuteScan.go

```diff
@@ -29,7 +29,7 @@ func checkmarxExecuteScan(config checkmarxExecuteScanOptions, telemetryData *tel
 	client := &piperHttp.Client{}
 	options := piperHttp.ClientOptions{MaxRetries: config.MaxRetries}
 	client.SetOptions(options)
-	sys, err := checkmarx.NewSystemInstance(client, config.ServerURL, config.Username, config.Password)
+	sys, err := checkmarx.NewSystemInstance(client, config.ServerURL, config.Username, config.Password, config.ClientID, config.ClientSecret)
```

Comment:
Would be cool if interfaces in Go could not only be used to specify methods, but also public fields. Then NewSystemInstance() could specify such an interface type as parameter, and "config" would satisfy it. But I digress.